convert-hf-to-gguf.py

```diff
@@ -218,6 +218,8 @@ def from_model_architecture(model_architecture):
             return BertModel
         if model_architecture == "NomicBertModel":
             return NomicBertModel
+        if model_architecture in "GemmaForCausalLM":
```

Comment:
Shouldn't this be
```suggestion
        if model_architecture == "GemmaForCausalLM":
```
unless you actually do mean to do a substring search here?